### PR TITLE
issue 681 MAPD-361 geojson nested

### PR DESCRIPTION
**Pull request by nmccready**
- geojson nested working
- nested geojson specs

### CODE REVIEW

**Comment by nmccready** [comment]
@zacronos

File: src/directives/geojson.js [hunk 1]
```diff
@@ -58,19 +60,28 @@ angular.module("leaflet-directive")
...
                 };
 
+                var isNested = (hlp.isDefined(attrs.geojsonNested) &&
+                    hlp.isTruthy(attrs.geojsonNested));
+
                 var _clean = function(){
-                    if (isDefined(leafletGeoJSON) && map.hasLayer(leafletGeoJSON)) {
-                        map.removeLayer(leafletGeoJSON);
+                    var _remove = function(lObject) {
+                        if (isDefined(lObject) && map.hasLayer(lObject)) {
```
Comment (nmccready) [inline src/directives/geojson.js]: support to iterate over a nested structure.

File: examples/geojson-nested.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html ng-app="demoapp">
+<head>
+    <script src="../bower_components/angular/angular.min.js"></script>
```
Comment (nmccready) [inline examples/geojson-nested.html]: please look at this example vs geojson-non-nested.html

**Comment by nmccready** [comment]
@tombatossals

**Comment by zacronos** [comment]
LGTM

**Pull request merged**